src/proxy.rs

```diff
@@ -157,6 +157,31 @@ impl<'a> Proxy<'a> {
             .map_err(From::from)
     }
 
+    pub(crate) async fn signal_with_args<I>(
+        &self,
+        name: &'static str,
+        args: &[(u8, &str)],
```

Comment:
can't the args also be `[(u8, &'static str)]`?

Reply:
I don't understand what you mean, I don't think it's possible because array wouldn't have a known size at compile time. The dbus function `receive_signal_with_args` also takes a slice `&[(u8, &str)]`. I don't think it makes sense to make this function generic over different arraysizes.

Reply:
Never mind, I should have enough coffee when reviewing stuff next time